__init__.py

```diff
@@ -13,7 +13,7 @@
 
 bl_info = {
     "name": "Dream Textures",
-    "author": "Carson Katri",
+    "author": "Carson Katri, Kevin C. Burke",
```

Comment:
If this should include all contributors, can you include Greg Richardson (@gregnr)?

```suggestion
    "author": "Carson Katri, Greg Richardson, Kevin C. Burke",
```